### PR TITLE
lib: add address method to Server

### PR DESCRIPTION
**Pull request by lundibundi**
This allows to properly get server address parameters
without the need to break abstraction to get them from
low level server.

### CODE REVIEW

**Comment by belochub** [comment]
@lundibundi, you shouldn't add `test` label when you include tests connected to implemented functionality.

**Comment by lundibundi** [comment]
@belochub sure.

**Comment by aqrln** [comment]
Landed in 21ff8b7347d7de6bb3d06eb4e0c2e68ad2edc0d2.

File: test/node/get-server-address-ipc.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+'use strict';
```
Comment (aqrln) [inline test/node/get-server-address-ipc.js]: @belochub btw, after some thinking about it I'm now on your side about the fate of these tests if/when we'll be returning subclasses of Node.js core servers. Not that I'll be determined to remove them and I still think that extra tests aren't a bad thing even if there's not much need in them, but both keeping and removing these tests will sound fine to me.

**Pull request closed without merging**